RFC/src/RFC-0201_TariScript.md

```diff
@@ -482,11 +482,23 @@ Once again, transaction fees are ignored to simplify the illustration.
 Alice owns \\( C_a \\) and provides the required script to spend the UTXO as was described in the previous cases.
 
 Alice needs a public key from Bob, \\( K_{Sb} \\) to complete the one-sided transaction. This key can be obtained
-out-of-band, and will typically be Bob's wallet public key on the Tari network.
+out-of-band, and might typically be Bob's wallet public key on the Tari network.
+
+Bob requires the value \\( v_b \\) and blinding factor \\( k_b \\) to claim his payment, but he needs to be able to claim it without asking Alice for them.
+
+This information can be obtained by using Diffie-Hellman and Bulletproof rewinding. If the blinding factor \\( k_b \\) was calculated with diffie-hellman using the offset public keypair, (\\( k_{Ob} \\),\\( K_{Ob} \\)) as sender keypair and
+the keypair, (\\( k_{Sb} \\),\\( K_{Sb} \\)) as the receiver keypair, the blinding factor \\( k_b \\) can be securely calculated without communication.
 
 Alice uses Bob's public key to create a shared secret, \\( k_b \\) for the output commitment, \\( C_b \\), using
 Diffie-Hellman key exchange.
 
+Alice calculates \\( k_b \\) as
+$$
+    k_b = k_{Ob} * {K_Sb}
+$$
+
+Next Alice next uses Bulletproof rewinding to encrypt the value \\( v_b \\) into the the Bulletproof for the commitment \\( C_b \\). For this she uses (\\( k_{rewind} =  Hash(k_{b}) \\) as the encryption key.
```

Comment:
We need two private keys here: `pvt_rewind_key` and `pvt_blinding_key`. This could be something like:
```
k_{rewind} =  Hash(k_{b})
k_{blinding} =  Hash(Hash(k_{b} | "one-sided transaction"))
```
Please see the Bulletproofs code snippet for the full explanation.